### ISSUE TITLE
Helmfile charts deletion order is not correct if we have more than 6 files in sub-charts folder

### ISSUE THREAD

**Issue by mkruliv** [roboll/helmfile#1450]
Hello! 
Helmfile chart deletion (from the last chart to first) order is not correct if we have more than 6 files in sub-charts folder. For example, for 7 files we have the next deletion order - first file charts, next 6th. 5th .. 3. 2. and finally 7th.
Correct order should be 7,6,5,4,3,2,1
Correctly working only if we have less than 7 files.
Same situation if files are divided between subfolders.

helmfile version v0.125.5

Commands to run

```
# Creation
helmfile --file charts/helmfiles/helmfile.yaml --environment dev sync
# Deletion
helmfile --file charts/helmfiles/helmfile.yaml --environment dev destroy
```
Helmfile.yaml
```
bases:
  - common/helm-config.yaml
  - common/cluster_templates.yaml
---
helmfiles:
  - path: sub-helmfiles/01-boot.yaml
  - path: sub-helmfiles/02-core.yaml
  - path: sub-helmfiles/03-addons.yaml
  - path: sub-helmfiles/04-services.yaml
  - path: sub-helmfiles/05-backup.yaml
  - path: sub-helmfiles/06-test.yaml
  - path: sub-helmfiles/07-final.yaml
```
What could be the reason for this issue? Thanks.

**Comment by mumoshu** [comment]
@mkruliv Thanks for reporting. This is kind of embarrassing... but there was a terrible bug that resulted in random deletion order, rather than the reverse order. I've already submitted the fix at #1451, and it should be released as the part of upcoming Helmfile v0.126.2!

**Comment by mkruliv** [comment]
@mumoshu  Thanks for the help and for the quick response.
